.gitignore

```
@@ -115,4 +115,4 @@ dmypy.json
 .ci/e2e-test/ginkgo
 .ci/e2e-test/ssh_id
 
-/deploy/charts/rawfile-csi/Chart.lock
\ No newline at end of file
+/deploy/helm/rawfile-csi/helm.lock
```

Has the lock file been renamed somehow or was this a typo change?

```suggestion
/deploy/helm/rawfile-csi/Chart.lock
```

---

It was a typo